src/test/scala/nl/knaw/dans/easy/umd/CommandSpec.scala

```diff
@@ -0,0 +1,67 @@
+/**
```

Comment:
@jo-pol: We should really not create any more unit tests that depend on a specific environment, such as `deasy` being up. Keep the build (including the unit test) self-contained. Then we all know what it means when the build passes.

Please, either properly mock Fedora or just remove this test.